scripts/gen/preview-world-vscode.sh

```diff
@@ -17,21 +17,14 @@
 # preview with CTRL+PageUp, and then
 # CTRL+w will close the redundant image preview.
 
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-cd $SCRIPT_DIR/..
+cd $(git rev-parse --show-toplevel)
```

Comment:
Are you sure this is the correct directory?  Above had `$SCRIPT_DIR/..` .  Though it's possible that one became incorrect when this script was moved.

Reply:
I think this was just never fixed from when we made subdirs.

Reply:
I think you're right.